tests/unit/SidebarUtilsTest.ts

```diff
@@ -232,6 +233,46 @@ describe('SidebarUtils', () => {
             expect(optionDataPinned?.isPinned).toBe(true);
             expect(optionDataUnpinned?.isPinned).toBe(false);
         });
+
+        it('returns null when report is archived', async () => {
+            const MOCK_REPORT: Report = {
+                reportID: '5',
+            };
+
+            const reportNameValuePairs = {
+                private_isArchived: DateUtils.getDBTime(),
+            };
+
+            await Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${MOCK_REPORT.reportID}`, MOCK_REPORT);
+            await Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${MOCK_REPORT.reportID}`, reportNameValuePairs);
```

Comment:
```suggestion
            await Onyx.set(`${ONYXKEYS.COLLECTION.REPORT_NAME_VALUE_PAIRS}${MOCK_REPORT.reportID}`, reportNameValuePairs);
```

NAB. Just so we don't confuse values that were set in other tests (if any)